basiclibrary/src/main/java/basiclibrary/ArrayManipulation.java

```diff
@@ -0,0 +1,52 @@
+package basiclibrary;
+
+public class ArrayManipulation {
+    public int[] roll(int n){
+        int[] resultArr =  new int [n];
+        int min = 1;
```

Comment:
Great! You never want to have random number in your code, it's always a good idea to save numbers to a variable that describes what they are. It is convention that when you save a hard-coded value to an array, the variable name should be in all caps and any spaces should be replaced with _. In this case, min should be MIN and max should be MAX.